templates/parachain/README.docify.md

```diff
@@ -28,6 +28,7 @@
   - [Takeaways](#takeaways)
 
 - [Runtime development](#runtime-development)
+- [Deploy to Paseo TestNet & See Your First Block](#deploy-to-paseo-testnet--see-your-first-block)
```

Comment:
Not sure why but the link doesn't work. You can check if it works opening your branch after pushing a tentative fix: https://github.com/paritytech/polkadot-sdk/blob/seemant-updating-parachain-readme/templates/parachain/README.md#deploy-to-paseo-testnet--see-your-first-block.

Reply:
The link works like in the previous comment I had, but not like in your last update. Clicking on the section from ToC still keeps me in the same place.